src/test/java/com/epam/tc/hw2/ex1/TestForFirstExercise.java

```diff
@@ -0,0 +1,150 @@
+package com.epam.tc.hw2.ex1;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.time.Duration;
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+public class TestForFirstExercise {
+
+    WebDriver driver;
+    WebDriverWait wait;
+
+    @BeforeMethod
+    public void setUp() {
+        WebDriverManager.chromedriver().setup();
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        //Step 1. Open test site by URL
+        driver.get("https://jdi-testing.github.io/jdi-light/index.html");
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(5));
+        //set Explicity
+        wait = new WebDriverWait(driver, Duration.ofSeconds(10));
+
+    }
+
+    @AfterMethod
+    public void tearDown() {
+        //12. Close Browser
+        driver.quit();
+
+    }
+
+    @Test
+    public void testForFirstExercise() {
+
+        //Step 2. Assert Browser title
+        SoftAssert softAssert = new SoftAssert();
+        wait.until(ExpectedConditions.titleIs("Home Page"));
+        softAssert.assertEquals(driver.getTitle(), "Home Page");
+
+        //Step 3. Perform login
+        WebElement findDropDown = driver.findElement(By.className("navbar-right"));
+        wait.until(ExpectedConditions.elementToBeClickable(By.className("navbar-right")));
+        findDropDown.click();
```

Comment:
can be shortened
```suggestion
        wait.until(ExpectedConditions.elementToBeClickable(findDropDown)).click();
```

\+ same for other `wait.until(`, because this method returns element